InventorySalesAdminUi/view/adminhtml/web/template/product/form/fieldset.html

```diff
@@ -0,0 +1,57 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<div class="fieldset-wrapper"
+     css="$data.additionalClasses"
+     attr="'data-level': $data.level, 'data-index': index"
+     data-bind="visible: $data.visible === undefined ? true: $data.visible">
+    <div class="fieldset-wrapper-title"
+         attr="tabindex: !collapsible ? -1 : 0,
+               'data-state-collapsible': collapsible ? opened() ? 'open' : 'closed' : null"
+         click="toggleOpened"
+         keyboard="13: toggleOpened"
+         if="label">
+
+        <strong css="'admin__collapsible-title': collapsible,
+                      title: !collapsible,
+                      '_changed': changed,
+                      '_loading': loading,
+                      '_error': error">
+            <span translate="label"/>
+            <div class="admin__field-saleable-qty admin__field-tooltip">
+                <a class="admin__field-tooltip-action action-help" target="_blank" tabindex="1" attr="href: tooltip.link"/>
+                <div class="admin__field-tooltip-content" html="tooltip.description"/>
```

Comment:
Please, change to text binding instead of html since the value supposes to be text only.